flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/KinesisConfigUtil.java

```diff
@@ -66,27 +74,35 @@
 	 **/
 	protected static final String THREAD_POOL_SIZE = "ThreadPoolSize";
 
-	/** Default values for RateLimit. **/
+	/**
+	 * Default values for RateLimit.
+	 **/
 	protected static final long DEFAULT_RATE_LIMIT = 100L;
 
-	/** Default value for ThreadingModel. **/
+	/**
+	 * Default value for ThreadingModel.
+	 **/
 	protected static final KinesisProducerConfiguration.ThreadingModel DEFAULT_THREADING_MODEL = KinesisProducerConfiguration.ThreadingModel.POOLED;
 
-	/** Default values for ThreadPoolSize. **/
+	/**
+	 * Default values for ThreadPoolSize.
+	 **/
 	protected static final int DEFAULT_THREAD_POOL_SIZE = 10;
 
 	/**
 	 * Validate configuration properties for {@link FlinkKinesisConsumer}.
 	 */
-	public static void validateConsumerConfiguration(Properties config) {
+	public static void validateConsumerConfiguration(Properties config, List<String> streams) {
 		checkNotNull(config, "config can not be null");
 
 		validateAwsConfiguration(config);
 
+		validateEFOConfiguration(config, streams);
```

Comment:
@xiaolong-sn nit: Camelcase applies to acronyms too, typically `validateEFOConfiguration` should be `validateEfoConfiguration`. This applies to all instances of fields/methods in the PR